multi-node.sh

```diff
@@ -182,4 +182,18 @@ if [ ${BUILD_GDR} -eq 1 ]; then
     fi
     set -e
 fi
+
+# Run cdi_test
+if [ ${RUN_CDI_TESTS} -eq 1 ]; then
```

Comment:
This PR implements the test, but has not enabled it yet. Is that correct?

Reply:
I have RUN_CDI_TEST set as 1 by default in multi-node tests.